<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Han's E05 pick and place control application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_robot">
      <property name="maximumSize">
       <size>
        <width>370</width>
        <height>180</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>MG400</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="btn_robot_Connect">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_robot_Control">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Show dashboard</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label_robot_Connect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>MG400 No Connection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_robot_IP">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Address: *</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>272</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionJapanese"/>
    <addaction name="actionVietnamese"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionJapanese">
   <property name="text">
    <string>日本語</string>
   </property>
  </action>
  <action name="actionVietnamese">
   <property name="text">
    <string>Tiếng Việt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
